SCOPE.md

```diff
@@ -0,0 +1,32 @@
+# AsyncAPI Scope
+
+AsyncAPI is a specification for defining message-driven APIs. We're on a mission to standardize message-based communication and increase interoperability of the different types of systems out there.
+
+## Scope
+
+The following is a list of features the AsyncAPI specification and its official tooling should offer:
+
+- Provide enough information to connect and consume message-driven APIs. An AsyncAPI document should provide all your software needs to connect to a given API and start publishing and/or consuming messages from it.
+- Offer a framework for validating message payloads and headers.
+- Allow you to strictly define your message-driven API in a machine-readable format.
+- Provide mechanisms for mapping the information contained in an AsyncAPI document to the major messaging protocols.
```

Comment:
mechanisms, or a definition language to describe protocols available for publishing and consuming messages?

Reply:
Both, a definition language and tools to process these definitions.

Reply:
To clarify is "official tooling" both that which is originated within, or donated to, the AsyncAPI github organisation and no others?

Reply:
So far yes. The ones the AsyncAPI is responsible for maintaining.